modules/rss.py

```diff
@@ -37,14 +31,22 @@ def _format_entry(self, server, channel, feed_title, entry, shorten):
                 pass
         link = "%s" % link if link else ""
 
-        feed_title_str = "%s" % feed_title if feed_title else ""
+        variables = dict(
+            longtitle=feed_title or "",
+            title=utils.parse.line_normalise(utils.http.strip_html(
+                entry["title"])),
+            link=link or "",
+            author=entry.get("author", "unknown author") or "",
+        )
```

Comment:
Let's merge the `variables` and `entry` dicts into one so we can just format them all as one.
```suggestion
        )
        variables.update(entry)
```

Reply:
This would change the behavior though. Let's keep this for another PR?

Reply:
The problem with that is the current behavior kinda sucks. I'd prefer to not have all this fragmentation in our formatting logic.